bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
+
```

Comment:
I *think* [the bug is saying](https://bugzilla.mozilla.org/show_bug.cgi?id=1354710#c5) they want to use this template as well. If so, we'd need the same change as applied above, as well as some view logic.

Though they could mean they want to use the standard firstrun template, but incorporate the FxA UI/UX from FC 99...which appears to be triggered via the `entrypoint` param? Not clear to me.